Dynavity/Dynavity/model/elements/CanvasElementProtocol.swift

```diff
@@ -0,0 +1,96 @@
+import Foundation
+import CoreGraphics
+
+// Conforming to `AnyObject` forces this to be a class-only protocol.
```

Comment:
I think this can be removed

Reply:
Oops, missed this out while experimenting. Thanks!

Reply:
It's fine!